src/connection.rs

```diff
@@ -21,11 +21,13 @@ impl Connection {
     fn init(dbus_manager: &Rc<DBusNetworkManager>, path: &str) -> Result<Self, String> {
         let settings = dbus_manager.get_connection_settings(path)?;
 
-        Ok(Connection {
-               dbus_manager: dbus_manager.clone(),
-               path: path.to_string(),
-               settings: settings,
-           })
+        Ok(
+            Connection {
```

Comment:
This one is caused by `fn_call_style = "Block"`, but improvements already landed in rustfmt master, so we will have better formatting for those cases soon.